packages/sdk-redux/src/redux-slices/rtk-query/getSerializeQueryArgs.ts

```diff
@@ -0,0 +1,49 @@
+import {isPlainObject} from '@reduxjs/toolkit';
+import {SerializeQueryArgs} from '@reduxjs/toolkit/dist/query/defaultSerializeQueryArgs';
+
+export const getSerializeQueryArgs =
+    (): SerializeQueryArgs<any> =>
+    ({endpointName, queryArgs}) => {
+        // NOTE: The code below is taken from Redux Toolkit's repository from `defaultSerializeQueryArgs.ts`.
+
+        // Comment from RTK-Query: Sort the object keys before stringifying, to prevent useQuery({ a: 1, b: 2 }) having a different cache key than useQuery({ b: 2, a: 1 })
+        return `${endpointName}(${JSON.stringify(queryArgs, (_key, value) =>
+            isPlainObject(value)
+                ? Object.keys(value)
+                      .sort()
+                      .reduce<any>((acc, key) => {
+                          acc[key] = normalizeValue((value as any)[key]);
+                          return acc;
+                      }, {})
+                : normalizeValue(value)
+        )})`;
+    };
+
+// TODO(KK): The logic below is duplicated in SDK-Core. Think of a way of sharing it.
+
+// NOTE: Regex taken from Ethers.
+const isAddressRegex = /^(0x)?[0-9a-fA-F]{40}$/;
```

Comment:
reuse it?

Reply:
I'll leave it as it is for now as I don't have a very clear idea of the best way of re-using it. I'll tackle it next time I come across it.